NativeApp/src/components/Common/HeaderDays/styles.js

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components';
+import colours from '../../../utilities/globalStyles/theme';
+
+export const HeaderContainer = styled.View`
```

Comment:
Would prefer if we removed `style-components` but for the time being add `marginTop: 50` to the `View`. The header is currently covering the options on iPhoneX.

Reply:
done